src/model/model.nit

```diff
@@ -570,7 +570,7 @@ class MClass
 	# Is `self` and abstract class?
 	var is_abstract: Bool is lazy do return kind == abstract_kind
 
-	redef fun mdoc_or_fallback do return intro.mdoc_or_fallback
+	redef fun mdoc_or_fallback do return intro.mdoc
```

Comment:
This is done in order to avoid an infinite recursion since the `MClassDef` rolls back to the documentation of its `MClass`.